static/templyte/src/app/download-template/download-template.component.css

```diff
@@ -2,3 +2,43 @@
     font-size:1.2em;
     font-family: 'Playfair Display', serif;
 }
+
```

Comment:
Make spacing consistent. (I recommend leaving one line between items, as is standard)